issue_finder.py

```diff
@@ -0,0 +1,62 @@
+from fnmatch import fnmatch
+from http.client import IncompleteRead
+import os
+import pathlib
+from urllib.error import HTTPError
+import urllib.request
+import yaml
```

Comment:
need to add requirements.txt file (or Pipfile if you're using pipenv) for pyyaml? 

not part of this commit but perhaps we should also add some python version > 3.6  in requirements.txt file since f strings not supported in python3 versions prior to 3.6

Reply:
Agree